<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorPage</class>
 <widget class="QWidget" name="EditorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="Splitter_">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QsciScintilla" name="TextEditor_">
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="ConsoleBox_">
      <property name="title">
       <string>Console</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPlainTextEdit" name="Console_"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="Inject_">
             <property name="text">
              <string>Inject</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Release_">
             <property name="text">
              <string>Release</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="ActionNew_">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="ActionIcon" stdset="0">
    <string notr="true">documentnew</string>
   </property>
  </action>
  <action name="ActionOpen_">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="ActionIcon" stdset="0">
    <string notr="true">documentopen</string>
   </property>
  </action>
  <action name="ActionSave_">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="ActionIcon" stdset="0">
    <string notr="true">documentsave</string>
   </property>
  </action>
  <action name="ActionSaveAs_">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="ActionIcon" stdset="0">
    <string notr="true">documentsaveas</string>
   </property>
  </action>
  <action name="ActionShowEOL_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show end of lines</string>
   </property>
  </action>
  <action name="ActionShowCaretLine_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show caret line</string>
   </property>
  </action>
  <action name="ActionWSInvisible_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invisible</string>
   </property>
  </action>
  <action name="ActionWSVisible_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always visible</string>
   </property>
   <property name="toolTip">
    <string>Always visible</string>
   </property>
  </action>
  <action name="ActionWSVisibleAfterIndent_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Visible after indent</string>
   </property>
  </action>
  <action name="ActionAutoIndent_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic indentation</string>
   </property>
  </action>
  <action name="ActionShowLineNumbers_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show line numbers</string>
   </property>
  </action>
  <action name="ActionEnableFolding_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable folding</string>
   </property>
  </action>
  <action name="ActionWrapNone_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No wrap</string>
   </property>
  </action>
  <action name="ActionWrapWords_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Word</string>
   </property>
  </action>
  <action name="ActionWrapCharacters_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Character</string>
   </property>
   <property name="toolTip">
    <string>Character</string>
   </property>
  </action>
  <action name="ActionReplace_">
   <property name="text">
    <string>Replace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
